.yamato/2DSpaceShooter_triggers.yml

```diff
@@ -0,0 +1,48 @@
+{% metadata_file .yamato/project.metafile %}
+---
+
+# Run all relevant tasks when a pull request targets the develop / main or a sample branch
+pull_request_trigger:
+  name: 2DSpaceShooter Pull Request Trigger (main, develop, & sample branches)
+  dependencies:
+{% for project in projects -%}
+{% if project.name =="2dspaceshooter" -%}
+{% for editor in project.test_editors -%}
+{% for platform in test_platforms -%}
```

Comment:
there's a lot of repetition in this part across samples triggers, where the only difference is `{% if project.name =="<project name>" -%}`
is there any way we can avoid repeating all this code several times? Maybe by adding a cname check in the "paths" section instead? Or it is impossible due to Yamato's nature?

Reply:
Yes, it is going through all projects and running further steps only if it is a specific project, that is reason I added if condition at line 9 , maybe there is a way , let me check, but for now this seems to be working for our purposes, it only starts jobs specific to the project.